### ISSUE TITLE
[Assignment]Homework#1 - ECE5605_41

### ISSUE THREAD

**Issue by 2hyel05** [JonghyeokPark/SWE3003-41-and-ECE5605-41#11]
I have a question for 'Homework#1 -Page Mapping FTL'. Is this instruction have only one page? I was wondering it has more instruction pages. The bottom of the page shows 1/28, but it shows me only one page of instruction. if so, what I have to do is just 'calculate the following metrics' part and explain why? I also would like to know the submission format too.

**Comment by Van-NguyenPham** [comment]
Dear TAs,
I would like to know what happens to garbage collection if there is no invalid page in all blocks?
Thank you.

**Comment by JonghyeokPark** [comment]
@2hyel05
First of all, you can ignore the total page number 28. Sorry for the typo. 
Instruction for Homework#1 is a single page. 
There's no strict rule for the submission format; for instance, write down your answer for "calculating the following metrics" and submit the  PDF file.

**Comment by JonghyeokPark** [comment]
@Nguyen-K60 
I'm not sure if I understand your question correctly. 
If there is no invalid page in all blocks when the GC is triggered, it means that SSD is (already) running out of space.
It won't happen in commercial SSDs because the space manager prevents this issue.

**Comment by Nguyen-K60** [comment]
@JonghyeokPark 
Yes, that is what I want to ask. I face this issue when solving homework #1. I don't know my solution is correct or not.
